prime-router/src/main/kotlin/cli/LookupTableCommands.kt

```diff
@@ -42,18 +42,20 @@ import java.time.Instant
 
 /**
  * Utilities to submit and get data from the Lookup Tables API.
+ * If a [useThisToken] is not specified, then attempt to get a token from Otka.
+ * Otherwise, [useThisToken] is sent as the bearer token to the ReportStream server.
  */
-class LookupTableEndpointUtilities(val environment: Environment) {
+class LookupTableEndpointUtilities(val environment: Environment, val useThisToken: String? = null) {
```

Comment:
Added optional ability to pass in a token so I could use this code in our smoke tests, with either an Okta or a Server2Server token.